javascript-carousel/main.js

```diff
@@ -0,0 +1,82 @@
+var $chevronLeft = document.querySelector('.fa-chevron-left');
+var $chevronRight = document.querySelector('.fa-chevron-right');
+var $img = document.querySelector('img');
+var $dotContainer = document.querySelector('.dots');
+var $dots = document.querySelectorAll('.fa-circle');
+var imageArr = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var counter = 0;
+var intervalId = null;
+
+function handlePrevImage(event) {
+  if (counter > 0) {
+    counter--;
+    imageAttr();
+    dotMarker();
+  } else {
+    counter = 5;
+    counter--;
+    imageAttr();
+    dotMarker();
+  }
+}
+
+function handleNextImage(event) {
+  if (counter < 4) {
+    counter++;
+    imageAttr();
+    dotMarker();
```

Comment:
Same revision here, but in reverse. You would now check if the value is greater than or equal to 4